packages/fxa-components/Survey/index.test.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { render, cleanup } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+import Survey, { CreateHandleIframeTask } from "./index";
+
+const surveyURL = "https://my-survey-url.mozilla.org/";
+
+afterEach(cleanup);
+
+/*
+ - render minimized
+ - expand / collapse
+ - complete and remove
```

Comment:
Are these todo items? Would be good to have these tests in (maybe #4769 shouldn't be closed just yet?).

Reply:
These are all done. I think that tests for closing and removing are out of the scope for this. I think they make more sense for testing in the parent component for the surveys. `App.js` in payments server